rkt/trust.go

```diff
@@ -59,7 +59,15 @@ func init() {
 func runTrust(cmd *cobra.Command, args []string) (exit int) {
 	if flagPrefix == "" && !flagRoot {
 		if len(args) != 0 {
-			stderr.Print("--root required for non-prefixed (root) keys")
+			trustName := args[0]
+			cmdName := "trust"
+			
+			stdout.Printf(`Prevented potentially dangerous unbounded trust of all containers signed by keys from %s
+To continue trusting this root domain:
+	rkt %s --root %s PUBKEY (local key file, no discovery)
+Or, establish a smaller scope with a prefix:
+	rkt %s --prefix "example.com/hello"
```

Comment:
I just tested it with a fresh build of the latest master:
```
matthiasjahn@ubuntu:~/rkt$ ./build-rkt-1.7.0+git/bin/rkt trust --prefix coreos.com/etcd
pubkey: prefix: "coreos.com/etcd"
key: "https://coreos.com/dist/pubkeys/aci-pubkeys.gpg"
gpg key fingerprint is: 8B86 DE38 890D DB72 9186  7B02 5210 BD88 8818 2190
	CoreOS ACI Builder <release@coreos.com>
Are you sure you want to trust this key (yes/no)?
^C
matthiasjahn@ubuntu:~/rkt$ ./build-rkt-1.7.0+git/bin/rkt trust --prefix coreos.com/etcd https://coreos.com/dist/pubkeys/aci-pubkeys.gpg
pubkey: prefix: "coreos.com/etcd"
key: "https://coreos.com/dist/pubkeys/aci-pubkeys.gpg"
gpg key fingerprint is: 8B86 DE38 890D DB72 9186  7B02 5210 BD88 8818 2190
	CoreOS ACI Builder <release@coreos.com>
Are you sure you want to trust this key (yes/no)?
^C
matthiasjahn@ubuntu:~/rkt$ ./build-rkt-1.7.0+git/bin/rkt trust --prefix coreos.com/etcd aci-pubkeys.gpg
pubkey: prefix: "coreos.com/etcd"
key: "aci-pubkeys.gpg"
gpg key fingerprint is: 8B86 DE38 890D DB72 9186  7B02 5210 BD88 8818 2190
	CoreOS ACI Builder <release@coreos.com>
Are you sure you want to trust this key (yes/no)?
^C
matthiasjahn@ubuntu:~/rkt$ ./build-rkt-1.7.0+git/bin/rkt trust --root aci-pubkeys.gpg
pubkey: prefix: ""
key: "aci-pubkeys.gpg"
gpg key fingerprint is: 8B86 DE38 890D DB72 9186  7B02 5210 BD88 8818 2190
	CoreOS ACI Builder <release@coreos.com>
Are you sure you want to trust this key (yes/no)?
^C
matthiasjahn@ubuntu:~/rkt$ ./build-rkt-1.7.0+git/bin/rkt trust --root https://coreos.com/dist/pubkeys/aci-pubkeys.gpg
pubkey: prefix: ""
key: "https://coreos.com/dist/pubkeys/aci-pubkeys.gpg"
gpg key fingerprint is: 8B86 DE38 890D DB72 9186  7B02 5210 BD88 8818 2190
	CoreOS ACI Builder <release@coreos.com>
Are you sure you want to trust this key (yes/no)?
^C
```

So it seems, with `--prefix` it is possible without a PUBKEY (auto-discovery) or with a local PUBKEY-file or with a PUBKEY-file via URL. With `--root` it has to be a PUBKEY - either local or via URL. Does it mean that the last line: `To trust a key for an entire root domain, you must use the --root flag, with a path to a local key file (no discovery)` in https://github.com/coreos/rkt/blob/master/Documentation/subcommands/trust.md#rkt-trust should be changed, since it says `with a path to a local key file` but a URL is also possible? Then i can change that also in this PR if you want.

My suggestion now:
```
trust: Prevented potentially dangerous unbounded trust of all containers signed by keys from coreos.com
To continue trusting a specific prefix of this domain:
    rkt trust --prefix "coreos.com/foo" [PUBKEY]
    rkt trust --prefix "coreos.com/foo/*" [PUBKEY]
Or, trust the root domain:
    rkt trust --root PUBKEY
```

Well, i hope i am right. @lucab what do you think?